### ISSUE TITLE
It doesn't like my Java datapack files

### ISSUE THREAD

**Issue by tryashtar** [Blockception/VSCode-Bedrock-Development-Extension#564]
I think the extension is overly eager to appear; it puts warnings whenever I'm looking at certain Java datapack files.

![image](https://github.com/Blockception/VSCode-Bedrock-Development-Extension/assets/26075577/6667b5d8-ebd0-4718-87ad-0b830a423e74)

I get these warnings for anything in the `particles` or `recipes` folders, and any file named `blocks.json` or `sounds.json`.

**Comment by DaanV2** [comment]
Don't think I can do much in the plugin itself.

You can deactivate the extension for the workspace. Or use the .mcignore file in the root folder to specify what to skip

Example: https://github.com/Blockception/VSCode-Bedrock-Development-Extension/blob/main/.mcignore

**Comment by tryashtar** [comment]
Sadly disabling it in a workspace isn't feasible for me :(
Is it out of the question to look for `manifest.json` to figure out where we are? This naive approach that just looks at a file or folder name causes other issues too, like if you make an entity named `blocks.json` all the warnings are wrong

**Comment by Kesuaheli** [comment]
+1

Would be very helpful to only get highlightings, autocompletions, warnings and errors when the behaviour pack or resource pack root folder contains a (valid?) `manifest.json` file.

Apart from colliding with JE data packs, I noticed the current approach also collides with other plain JSON files. Even when not on a Minecraft context at all.
A while ago I started to have this extention always disabled. Only enabling it when needed and immediately disable it globally again. I hope I dont have too tell you that this is not the way it should works, even so though it works ;)
